doc/whats_new/v1.2.rst

```diff
@@ -155,6 +155,9 @@ Changelog
 - |Enhancement| :func:`datasets.dump_svmlight_file` is now accelerated with a
   Cython implementation, providing 2-4x speedups.
   :pr:`23127` by :user:`Meekail Zain <micky774>`
+  
+- |Enhancement| Path-like objects, such as those created with pathlib are now allowed as paths in :func:`datasets.load_svmlight_file` and :func:`datasets.load_svmlight_files`.
```

Comment:
May this be wrapped to be less than 88 characters per line to match the rest of the file?